packages/terra-tabs/docs/README.md

```diff
@@ -0,0 +1,167 @@
+# Terra Tabs
+
+Tabs are containers used to organize content. They allow for quick switching between groups of contextually related content. Content is divided into different containers and each container is viewable one at a time. The user can switch between containers by selecting the corresponding tab control.
+
+Tabs are meant for organizing content that doesn’t need to be compared or accessed simultaneously or if the user needs quick access through a group of contextually related content. Tabs are not intended for navigation.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-tabs`
+
+## Usage
+
+### Note
+The tabs component will apply the prop `isLabelHidden` to all components passed as content for Tabs.Pane. This will indicate if any of the tabs are icon only or have been truncated. When this prop is true a header should be rendered in the content area to for accessibility purposes.
+
+### Example Tab Content Component
+
+```jsx
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  isLabelHidden: PropTypes.bool,
+  label: PropTypes.string,
+  children: PropTypes.node,
+};
+
+
+const TabContent = ({ isLabelHidden, label, children }) => (
+  <div style={{ height: '200px', padding: '10px' }}>
+    {isLabelHidden ? <h3>{label}</h3> : null}
```

Comment:
Given the note above regarding `isLabelHidden` will consumers need to toggle isLabelHidden or is this handled by the Tab Content itself?

Reply:
The Tabs component will clone children elements passed to the Tab.Panes and apply the isLabelHidden prop. It will be true if there are any isIconOnly Panes or if any of the tabs have truncated labels. The consumer only needs to know that that they should display a header if `isLabelHidden` is true.